### PR TITLE
Kd feature videos

### PR DESCRIPTION
**Pull request by KieranDevlin**

### CODE REVIEW

File: client/js/components/SpeakerCard/SpeakerCard.js [hunk 1]
```diff
@@ -4,8 +4,17 @@ import styles from './styles';
 import {TouchableOpacity} from 'react-native-gesture-handler';
 import Text from '../CustomText/CustomText';
 
-const SpeakerCard = ({name, style, source}) => {
-  return (
+const SpeakerCard = ({name, style, source, route}) => {
```
Comment (siddharthparmar7) [inline client/js/components/SpeakerCard/SpeakerCard.js]: your props can be just one object.

File: client/js/components/SpeakerCard/SpeakerCard.js [hunk 2]
```diff
@@ -4,8 +4,17 @@ import styles from './styles';
...
+  return route?.name === 'Video' ? (
+    <TouchableOpacity style={[styles.button, style]}>
+      <Image
+        style={{height: '100%', width: '25%'}}
+        resizeMode={'cover'}
+        source={source}
```
Comment (siddharthparmar7) [inline client/js/components/SpeakerCard/SpeakerCard.js]: move your styles to styles.js

File: client/js/components/SpeakerCard/SpeakerCard.js [hunk 3]
```diff
@@ -4,8 +4,17 @@ import styles from './styles';
...
+      />
+      <Text style={styles.videoLabel}>{name}</Text>
+    </TouchableOpacity>
+  ) : (
     <TouchableOpacity style={[styles.button, style]}>
       <Image
         style={{height: '75%', width: '100%'}}
```
Comment (siddharthparmar7) [inline client/js/components/SpeakerCard/SpeakerCard.js]: same as above

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice work! Please make the changes.

File: client/js/components/SpeakerCard/SpeakerCard.js (hunk 1 above)
Comment (KieranDevlin) [reply client/js/components/SpeakerCard/SpeakerCard.js]: This is just dummy data for now, can we reconfigure props when we start working on the db queries?

**Review by siddharthparmar7 — APPROVED** [review]

**Comment by siddharthparmar7** [comment]
Please resolve merge conflict.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
I can merge it as soon as you resolve merge conflicts. Thanks

**Pull request merged**